### PR TITLE
Airdrop setup

### PR DESCRIPTION
**Pull request by ecmendenhall**
Now using the Balancer [`MerkleOrchard`](https://docs.balancer.fi/products/merkle-orchard) 🌳 🌳 🌳 

- Add an `airdrop:create` task to create a distribution using the Balancer Merkle Orchard. See the README in the task directory for more docs on how to use it.
- Add the `merkleTree.ts` library used by Balancer. (Re-use this to generate proofs).
- Add a fork test that creates and tests a Merkle Orchard distribution.
- Add Arbitrum + Polygon and testnets to Hardhat config.

### CODE REVIEW

**Review by anthonymartin — APPROVED** [review]
looks good, made a couple minor comments

File: packages/hardhat/lib/tasks/airdrop/create.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  "arbitrum",
+  "arbitrumRinkeby",
+];
+
+async function main(args: Args, hre: HardhatRuntimeEnvironment) {
+  if (!SUPPORTED_NETWORKS.includes(hre.network.name)) {
+    throw new Error(
+      `Unsupported network. The selected network is: ${hre.network.name}`
+    );
+  }
```
Comment (ecmendenhall) [inline packages/hardhat/lib/tasks/airdrop/create.ts:35]:
```suggestion
```

File: packages/hardhat/lib/tasks/airdrop/create.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+const SUPPORTED_NETWORKS = [
+  "localhost",
+  "mainnet",
+  "kovan",
+  "rinkeby",
+  "polygon",
+  "mumbai",
+  "arbitrum",
+  "arbitrumRinkeby",
+];
```
Comment (ecmendenhall) [inline packages/hardhat/lib/tasks/airdrop/create.ts]:
```suggestion
```

File: packages/hardhat/lib/tasks/airdrop/create.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+const SUPPORTED_NETWORKS = [
+  "localhost",
+  "mainnet",
+  "kovan",
+  "rinkeby",
+  "polygon",
+  "mumbai",
+  "arbitrum",
+  "rinkarby",
+];
```
Comment (ecmendenhall) [inline packages/hardhat/lib/tasks/airdrop/create.ts:28]:
```suggestion
```

**Pull request merged**